rpgs/views.py

```diff
@@ -131,6 +137,22 @@ def test_func(self):
         return can_manage(self.request.user.member, rpg)
 
 
+class Notify(LoginRequiredMixin, generic.View):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def post(self, request, *args, **kwargs):
+        rpg = get_object_or_404(Rpg, pk=self.kwargs['pk'])
+
+        if not can_manage(self.request.user.member, rpg):
```

Comment:
This is probably fine, but I'd recommend instead using a UserPassesTestMixin and putting these checks into test_func

Reply:
Ignoring only because I _think_ this would require two `get_object_or_404` calls, which seems a bit silly. But also it's nearly 11pm sooooooooooooo

Reply:
Change of heart thanks to further conversation, resolved via 6f2b191